lib/takeaway.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'send_sms'
+
+class TakeAway
+
+  def initialize
+    @menu = { "chips" => 2.5, "kebab" => 5.5, "burger" => 4.75 }
```

Comment:
What if you wanted to create a new restaurant with a different menu?

Reply:
Or perhaps, several different restaurants, each with a different menu?

Reply:
Yes, I would perhaps have done this a different (better) way with a separate menu class or the like.